library/src/scala/quoted/Quotes.scala

```diff
@@ -238,6 +238,13 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
        *  This will be true when the macro is used in a transparent inline.
        */
       def isWhileTyping: Boolean
+
+      /**
+       * expose macro-specific settings as a list of strings.
+       * Settings can be set from command line with help of -Xmacro-settings opions.
```

Comment:
```
/** Expose macro-specific settings as a list of strings.
 *  Settings can be set from the command line with help of -Xmacro-settings options.
 *
 *  These will be used to expand any transparent macros or any non-transparent macro that is forced to expand while expanding the transparent macro.
 *  Non-transparent macros are not guaranteed to be expanded with the same set of settings.
 */
```

Reply:
i.e. transparent macroses can change settings?   Or it can be expanded during other compiler runs with potentially other settings?

Reply:
One case is if we compile the library with a set of settings and then recompile the from TASTy with other settings.